janis_bioinformatics/tools/common/bwaaligner.py

```diff
@@ -2,18 +2,49 @@
 import operator
 from janis_core import Array
 from janis_bioinformatics.data_types import FastqGzPair, FastaWithDict
-from janis_bioinformatics.tools import BioinformaticsWorkflow
+from janis_bioinformatics.tools import BioinformaticsWorkflow, BioinformaticsTool
 from janis_bioinformatics.tools.common.bwamem_samtoolsview import BwaMem_SamToolsView
 from janis_bioinformatics.tools.cutadapt import CutAdapt_2_1
 from janis_bioinformatics.tools.gatk4 import Gatk4SortSam_4_1_2
-from janis_bioinformatics.data_types.bam import Bam
+from janis_bioinformatics.data_types.bam import Bam, BamBai
 
 from janis_core.tool.test_classes import (
     TTestPreprocessor,
     TTestExpectedOutput,
     TTestCase,
 )
 
+"""
+def basic_test(bam_size, bai_size, flagstat, value):
```

Comment:
Let's add parameter type here

Reply:
also let's make the parameter `value` clearer. I assume that is the expected Bam file path?

Reply:
I'm sorry. This is some leftover from a very early stage. I should have deleted it.